cmd/wallet-web/src/pages/Signin.vue

```diff
@@ -49,14 +49,27 @@
             {{ t('Signin.heading') }}
           </span>
         </div>
-        <div class="grid grid-cols-1 gap-5 w-full sm:px-32 h-64 mb-12 items-center content-center">
+        <div
+          class="
+            grid grid-cols-1
+            gap-5
+            w-full
+            sm:px-32
+            h-64
+            mb-12
+            items-center
+            content-center
+            justify-items-center
```

Comment:
This is not a tailwind class

Reply:
I assume what you wanted is `items-center`, which is already there two lines above. However, it only work inside flex display, this container has grid

Reply:
yes i need grid and using flex here gives Classnames grid, flex are conflicting errors